### PR TITLE
Add ability to save event values

### PR DESCRIPTION
**Pull request by willfulbard**

### CODE REVIEW

File: server/camphoric/models.py [hunk 1]
```diff
@@ -76,7 +76,7 @@ class Event(TimeStampedModel):
         help_text="JsonLogic Registration-level pricing components")
 
     confirmation_page_template = models.TextField(blank=True, default='', help_text="Mustache template")
-    confirmation_email_subject = models.CharField(default='', max_length=100)
+    confirmation_email_subject = models.CharField(blank=True, default='', max_length=100)
```
Comment (evinism) [inline server/camphoric/models.py:79]: This is an unrelated change and should probably be its own PR

File: client/src/hooks/Provider.tsx [hunk 2]
```diff
@@ -92,8 +105,29 @@ export default class StateProvider extends React.Component<Props, State> {
     return value;
   }
 
+  async apiPut<P extends MinimumApiObject>(endpoint: ApiEndpoint, value: P): Promise<void> {
```
Comment (evinism) [inline client/src/hooks/Provider.tsx:108]: Part of me wants this to not be called "endpoint", and instead something like "resource" -- I think of endpoint as a full path rather than a parameter to a path.

Comment (willfulbard) [reply client/src/hooks/Provider.tsx:108]: See the latest, changed all endpoint to resource

File: server/camphoric/models.py (hunk 1 above)
Comment (willfulbard) [reply server/camphoric/models.py]: Removed, I assigned the issue to @joshuasorkin to fix in a separate MR

**Review by idelizer — APPROVED** [review]
